pinot-common/src/main/java/com/linkedin/pinot/common/config/TenantConfig.java

```diff
@@ -26,12 +26,12 @@
 public class TenantConfig {
   private static final Logger LOGGER = LoggerFactory.getLogger(TenantConfig.class);
 
-  @ConfigKey("brokerTenant")
-  @ConfigDoc("Broker tenant used by this table")
+  @ConfigKey("brokerTag")
+  @ConfigDoc("Broker tag used by this table")
   private String broker;
 
-  @ConfigKey("serverTenant")
-  @ConfigDoc("Server tenant used by this table")
+  @ConfigKey("serverTag")
+  @ConfigDoc("Server tag used by this table")
```

Comment:
Are we going to start keeping the whole tag name here (with suffix)? My understanding was that tag=<tenant_name>_<suffix>